pkg/gardenlet/apis/config/types.go

```diff
@@ -270,9 +268,9 @@ type ResourcesConfiguration struct {
 	Reserved corev1.ResourceList
 }
 
-// LeaderElectionConfiguration defines the configuration of leader election
+// GardenletLeaderElectionConfiguration defines the configuration of leader election
 // clients for components that can run with leader election enabled.
-type LeaderElectionConfiguration struct {
+type GardenletLeaderElectionConfiguration struct {
```

Comment:
Because it breaks the protobuf code generation, since there are 2 different types named `LeaderElectionConfiguration` (one is outside Gardener).

Reply:
Hmm, that's interesting.

Reply:
Since we decided to use `runtime.RawExtension`, protobuf generation for `GardenletConfiguration` is no longer needed. I will remove the tags and rename this type back to its original name.